### PR TITLE
swan-cern: Update gpu-operator to version v24.6.2

### PR DESCRIPTION
**Pull request by PMax5**

### CODE REVIEW

File: swan-cern/values.yaml [hunk 1]
```diff
@@ -268,29 +268,30 @@ fluentd:
...
+    version: v24.6.2
     defaultRuntime: containerd
     initContainer:
       repository: registry.cern.ch/kubernetes
       version: 11.4.2-base-ubi8
   driver:
     repository: registry.cern.ch/kubernetes
     image: nvidia-gpu-driver
-    version: "v550.54.15-6.6.13-200.fc39.x86_64"
+    version: "v550.54.15-6.8.4-200.fc39.x86_64"
```
Comment (etejedor) [inline swan-cern/values.yaml:285]: Can we have this as a separate commit, since it is not related to the gpu-operator update?

File: swan-cern/values.yaml [hunk 2]
```diff
@@ -268,29 +268,30 @@ fluentd:
...
     imagePullPolicy: Always
     manager:
       repository: registry.cern.ch/kubernetes
       version: v0.6.0
     licensingConfig:
       configMapName: "nvidia-grid-license"
-    kernelModuleConfig:
-      name: kernel-module-params
+      nlsEnabled: false
+  nfd:
```
Comment (etejedor) [inline swan-cern/values.yaml:293]: These changes are also unrelated to the update to fix the security issue? And also others below.

Comment (PMax5) [reply swan-cern/values.yaml:293]: It is to keep our chart in sync with the cern-magnum one. Should they be removed?

Comment (etejedor) [reply swan-cern/values.yaml:293]: What we can do, not to complicate things too much, is to separate this in two commits: one that updates the chart up until before the security issue fix, and one that strictly has the changes to fix the security issue.

File: swan-cern/values.yaml [hunk 3]
```diff
@@ -268,29 +268,30 @@ fluentd:
           path: ca-bundle.crt
 
 gpu-operator:
-  enabled: true
+  enabled: false
```
[the remark this answers is not in the document]
Comment (PMax5) [reply swan-cern/values.yaml]: Yes, I missed that but changed it now

File: swan-cern/values.yaml (hunk 2 above)
Comment (PMax5) [reply swan-cern/values.yaml]: Yeah that sounds good

**Review by etejedor — APPROVED** [review]
